Community restructure (#723)
iota-wiki/iota-wiki: internal/learn/the-community/how-to-support.md

---
title: How to support the vision
description: This page shows a list of ways to suport the vision
---

# How can you support the vision?

As ecosystem members there are different ways to support the vision.

This page is a non-exhaustive collection of ideas and smaller to bigger ways to support the growth and expansion of the ecosystem.

## Basic level

These activities require to register accounts on different platforms and time.

**First and foremost** join the IOTA Community on the IOTA Discord server. Click on [this link](http://discord.iota.org/) to get the invitation to the server and get in touch with the IOTA Foundation members and the awesome IOTA community members.

<!--
### Be a Tanglenaut

> An Tanglenaut supports the project on various social networks like [Twitter](https://twitter.com/iota/), [LinkedIn](https://www.linkedin.com/company/iotafoundation/) or platforms like [YouTube](https://www.youtube.com/c/iotafoundation), [GitHub](https://github.com/iotaledger/) and [Reddit](https://reddit.com/r/iota).

Reviewer: ```suggestion
A Tanglenaut supports the project on various social networks like [Twitter](https://twitter.com/iota/), [LinkedIn](https://www.linkedin.com/company/iotafoundation/) or platforms like [YouTube](https://www.youtube.com/c/iotafoundation), [GitHub](https://github.com/iotaledger/) and [Reddit](https://reddit.com/r/iota).
```

> Optional: Rank as *tanglenaut* on the IOTA Discord to be notified when news come out. To do so, join the **#get-your-badges** channel, type **!addrank tanglenaut** as message and send it. From now on if someone *pings* the Tanglenauts by typing **@Tanglenaut** in Discord you will get a notification.

Reviewer: ```suggestion
Optional: Rank as *Tanglenaut* on the IOTA Discord to be notified when news comes out. To do so, join the **#get-your-badges** channel, type **!addrank tanglenaut** as a message, and send it. From now on, if someone *pings* the Tanglenauts by typing **@Tanglenaut** in Discord, you will get a notification.
```

-->

Register an account <u>using a nickname</u> at the following sites and follow, subscribe, like, star, upvote the activities by the IOTA Foundation and the Ecosystem.

### Twitter

Liking and retweeting news in the ecosystem.

Difficulty level: **easy**

Time necessary: **15 minutes/week**

Sign up to Twitter: https://twitter.com/i/flow/signup

> Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow it daily to _like_ and _retweet_. Look for other projects or ecosystem members and follow them and like their tweets.

Reviewer: ```suggestion
Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to _like_ and _retweet_. Look for other projects or ecosystem members, follow them, and like their tweets.
```

---
title: How to support the vision
description: This page shows a list of ways to suport the vision
---

> # How can you support the vision?

Reviewer: Why would you write it in capital letters?

> # How can you support the vision?

Reviewer 2: Because those are our guidelines. Feel free to create a PR if you would like to change them :trollface: 
https://wiki.iota.org/learn/contribute-to-wiki/reference/style/formatting#headings

As ecosystem members there are different ways to support the vision.

This page is a non-exhaustive collection of ideas and smaller to bigger ways to support the growth and expansion of the ecosystem.

## Basic level

These activities require to register accounts on different platforms and time.

**First and foremost** join the IOTA Community on the IOTA Discord server. Click on [this link](http://discord.iota.org/) to get the invitation to the server and get in touch with the IOTA Foundation members and the awesome IOTA community members.

<!--
### Be a Tanglenaut

An Tanglenaut supports the project on various social networks like [Twitter](https://twitter.com/iota/), [LinkedIn](https://www.linkedin.com/company/iotafoundation/) or platforms like [YouTube](https://www.youtube.com/c/iotafoundation), [GitHub](https://github.com/iotaledger/) and [Reddit](https://reddit.com/r/iota).

Optional: Rank as *tanglenaut* on the IOTA Discord to be notified when news come out. To do so, join the **#get-your-badges** channel, type **!addrank tanglenaut** as message and send it. From now on if someone *pings* the Tanglenauts by typing **@Tanglenaut** in Discord you will get a notification.

> -->

Reviewer 3: As discussed in Discord, I want this to be a separate PR if the goal is to merge this at some point. If we merge comments like this we can't track the content changes anymore and have to find out who was responsible to ask what the status is. This is easier when it is just a draft PR, so we can properly track it.

> -->

Reviewer 2: Got moved. So this can be resolved

Follow the IOTA Foundation on LinkedIn [here](https://www.linkedin.com/company/iotafoundation/) by clicking on the three dots and then **+ Follow** to *like* and *share* the articles

![image-20210115161351884](./images/image-20210115161351884.png)

### YouTube

Watch the videos by the IOTA Foundation, learn about solutions built with IOTA and share them with the people you know.

Difficulty level: **easy**

Time necessary: **15 minutes/week**

Sign up to YouTube: https://www.youtube.com/signup

Subscribe to the IOTA Foundation channel [here](https://www.youtube.com/c/iotafoundation).

![image-20210115164439927](./images/image-20210115164439927.png)

### GitHub

Starring a repository has some potentially beneficial effects for the  repository. When a developer looks for a repository for certain  functionalities, the number of stars gives a rough indication about the  popularity and therefore the quality of the repository.

Difficulty level: **medium**

Time necessary: **30 minutes the first time, 5 minutes for every new repository (when they are published)**

Sign up to GitHub: https://github.com/join

> Then open the following repositories and *star* them.

Reviewer 2: Fixed with https://github.com/iota-wiki/iota-wiki/pull/723/commits/37bc73d81c32cc90a64bd57b9936da758678479c

![image-20210115162757470](./images/image-20210115162757470.png)

https://github.com/iotaledger/hornet  
https://github.com/iotaledger/firefly
https://github.com/iotaledger/iota.rs  
https://github.com/iotaledger/iota.c  
https://github.com/iotaledger/iota.go  
https://github.com/iotaledger/iota.js  

> https://github.com/iotaledger/stronghold.rs

Author: If you have a list that would be perfect!
Thanks

> https://github.com/iotaledger/stronghold.rs

Reviewer 4: I've updated the suggestion to include some more projects. I can't believe how that list keeps on growing :)

> https://github.com/iotaledger/stronghold.rs

Reviewer 2: Fixed with https://github.com/iota-wiki/iota-wiki/pull/723/commits/e9c98b123f2449165c7cc742715e793aedd0df7b

![image-20210115161351884](/img/participate/how-to-support/image-20210115161351884.png)

### YouTube

Watch the videos by the IOTA Foundation, learn about solutions built with IOTA and share them with the people you know.

- Difficulty level: **easy**.
- Time needed: **15 minutes/week**.
- Requirements: [Sign up to YouTube](https://www.youtube.com/signup) and
  [subscribe to the IOTA Foundation channel](https://www.youtube.com/c/iotafoundation).

![image-20210115164439927](/img/participate/how-to-support/image-20210115164439927.png)

### GitHub

Starring a repository has some potentially beneficial effects for the repository. When a developer looks for a repository for certain functionalities, the number of stars gives a rough indication about the popularity and therefore the quality of the repository.

- Difficulty level: **medium**.
- Time needed: **30 minutes the first time, 5 minutes for every new repository (when they are published)**.
- Requirements: [Sign up to GitHub](https://github.com/join).

> Then open the following repositories and _star_ them.

Reviewer 2: ```suggestion
After you have fulfilled the requirements, open the following repositories and *star* them:
```

![image-20210115162757470](/img/participate/how-to-support/image-20210115162757470.png)

> https://github.com/iotaledger/hornet  
> https://github.com/iotaledger/firefly  
> https://github.com/iotaledger/iota.rs  
> https://github.com/iotaledger/iota.c  
> https://github.com/iotaledger/iota.go  
> https://github.com/iotaledger/iota.js  
> https://github.com/iotaledger/stronghold.rs

Reviewer 2: ```suggestion
* [Firefly](https://github.com/iotaledger/firefly).
* [GoShimmer](https://github.com/iotaledger/goshimmer).
* [Hornet](https://github.com/iotaledger/hornet).
* [Identity](https://github.com/iotaledger/identity.rs).
* [iota.rs](https://github.com/iotaledger/iota.rs).
* [iota.c](https://github.com/iotaledger/iota.c ).
* [iota.go](https://github.com/iotaledger/iota.go).
* [iota.js](https://github.com/iotaledger/iota.js ).
* [IOTA 2.0 Research Specifications](https://github.com/iotaledger/IOTA-2.0-Research-Specifications).
* [IOTA Wiki](https://github.com/iota-wiki/iota-wiki).
* [Streams](https://github.com/iotaledger/streams).
* [Stronghold](https://github.com/iotaledger/stronghold.rs).
* [Wasp](https://github.com/iotaledger/wasp).
```

---
title: How to support the vision
description: This page shows a list of ways to suport the vision
---

> # How can you support the vision?

Reviewer 2: ```suggestion
# How Can You Support the Vision?
```

As ecosystem members there are different ways to support the vision.

This page is a non-exhaustive collection of ideas and smaller to bigger ways to support the growth and expansion of the ecosystem.

## Basic level

These activities require to register accounts on different platforms and time.

## Join the IOTA Discord

**First and foremost** join the IOTA Community on [the IOTA Discord](discord.md) server. Click on [this link](http://discord.iota.org/) to get the invitation to the server and get in touch with the IOTA Foundation members and the awesome IOTA community members.

## Social Media platforms

Register an account <u>using a nickname</u> at the following sites and follow, subscribe, like, star, upvote the activities by the IOTA Foundation and the Ecosystem.

### Twitter

#### Like and retweet news in the ecosystem.

- Difficulty level: **easy**
- Time needed: **15 minutes/week**
- Requirements: [Sign up to Twitter](https://twitter.com/i/flow/signup)

> Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to _like_ and _retweet_. Look for other projects or ecosystem members, follow them, and like their tweets.

Reviewer 2: ```suggestion
Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to *like* and *retweet*. Look for other projects or ecosystem members, follow them, and like their tweets.
```

> Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to _like_ and _retweet_. Look for other projects or ecosystem members, follow them, and like their tweets.

Reviewer 2: ```suggestion
Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to *like* and *retweet*. Look for other projects or ecosystem members, follow them, and like their tweets.
```

These are some [interesting IOTA related projects](<(https://twitter.com/i/lists/1564924491850989575)>) you can follow as well.

### LinkedIn

Like and reshare news from the IOTA Foundation and other ecosystem members with your professional network. Someone somewhere is looking for IOTA as a solution, and they do not know it yet.

- Difficulty level: **easy**.
- Time needed: **15 minutes/week**
- Requirements: [Sign up to LinkedIn](https://www.linkedin.com/signup/cold-join). You can also connect using your Google Account.

> Follow the IOTA Foundation on LinkedIn [here](https://www.linkedin.com/company/iotafoundation/) by clicking on the three dots and then **+ Follow** to _like_ and _share_ the articles

Reviewer 2: ```suggestion
Follow the IOTA Foundation on LinkedIn [here](https://www.linkedin.com/company/iotafoundation/) by clicking on the three dots and then **+ Follow** to *like* and *share* the articles
```

![image-20210115161351884](/img/participate/how-to-support/image-20210115161351884.png)

### YouTube

Watch the videos by the IOTA Foundation, learn about solutions built with IOTA and share them with the people you know.

- Difficulty level: **easy**.
- Time needed: **15 minutes/week**.
- Requirements: [Sign up to YouTube](https://www.youtube.com/signup) and
  [subscribe to the IOTA Foundation channel](https://www.youtube.com/c/iotafoundation).

![image-20210115164439927](/img/participate/how-to-support/image-20210115164439927.png)

### GitHub

> Starring a repository has some potentially beneficial effects for the repository. When a developer looks for a repository for certain functionalities, the number of stars gives a rough indication about the popularity and therefore the quality of the repository.

Reviewer 2: ```suggestion
Starring a repository has some potentially beneficial effects for the repository. When a developer looks for a repository for certain functionalities, the number of stars gives a rough indication of the popularity and, therefore, the quality of the repository.
```

Optional: Rank as *Tanglenaut* on the IOTA Discord to be notified when news comes out. To do so, join the **#get-your-badges** channel, type **!addrank tanglenaut** as a message, and send it. From now on, if someone *pings* the Tanglenauts by typing **@Tanglenaut** in Discord, you will get a notification.
-->
=======

> ## Socuial media platforms

Reviewer 2: Why are the merge indicators still here? 😅
